### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by joelfranusic-okta**

### CODE REVIEW

**Comment by joelfranusic-okta** [comment]
@yuliu-okta @cwu-okta @helenajerney-okta for your review.

@mystiberry-okta FYI

File: use_cases/api_security/index.html [hunk 1]
```diff
@@ -122,15 +122,15 @@ <h3 class="Sidebar-title">Use Cases</h3>
             
                 
             
-            <li class="is-active"><a href="/use_cases/api_security">API Access Management</a></li>
```
Comment (yuliu-okta) [inline use_cases/api_security/index.html]: Is this file auto generated from the index.md?

Comment (joelfranusic-okta) [inline use_cases/api_security/index.html]: It is generated from `index.md`, but not automatically. I had to generate it using `jekyll build`

Comment (joelfranusic-okta) [inline use_cases/api_security/index.html]: Ah, I see, you're asking why that got moved ... I'm not sure why it's indeterminate but have a fix in the works to make this placement deterministic.

**Pull request merged**